src/app/games/six/SixMinimax.ts

```diff
@@ -424,27 +441,29 @@ export class SixMinimax extends AlignementMinimax<SixMove,
             display(this.VERBOSE, '5+1 found!');
             // We found 5 pieces aligned and one space, so that space is a preVictory coord
             if (preVictory.isPresent() &&
-                preVictory.get().equals(lastEmpty) === false) {
+                (lastEmpty.isAbsent() || preVictory.equals(lastEmpty) === false)) {
```

Comment:
why the change ?

Reply:
car lastEmpty est optionnel, mais je propose de changer en preVictory.equals(lastEmpty)

Reply:
ah bah c'est le message plus bas :D